### ISSUE TITLE
qase-pytest is slowing down tests executions

### ISSUE THREAD

**Issue by KrishnaYadav2102** [qase-tms/qase-python#34]
with qase-pytest the execution time is 7 mins 33 secs
without qase-pytest the execution time is ~45 secs

is it possible to do bulk update test results once all the tests are executed?

QASE is amazing but this issue is holding me back to integrate it with my e2e.

Looking forward to quick fix here.

**Comment by Nickolasll** [comment]
I have simple test, just to try

```python
data = [1, 2, 3, 4, 6, 33]

@qase.id(*data)
@pytest.mark.parametrize('case', data)
def test_technical(case):
    print(case)
```

`Sequentially` tests, qase plugin `off` - 6 tests passed in 0.03s
`Sequentially` tests, qase plugin `on` - 6 tests passed in 11.53s
`Parallel` tests, qase plugin `off`, numprocesses = 6 - 6 tests passed in 0.56s
`Parallel` tests, qase plugin `on`, numprocesses = 6 - 6 tests passed in 64.59s

**Comment by n3r** [comment]
Will be fixed in v4.0.0.

**Comment by n3r** [comment]
Fixed in qase-pytest-4.0.0 #64
